### DISCUSSION TITLE
دریافت شماره کارت تراکنش

### DISCUSSION THREAD

**Discussion by FaridAghili** [evryn/laravel-toman#18] [Ideas]
سلام
امکان دریافت شماره کارت هم در زرین پال و هم در آی دی پی وجود داره
این پکیج این قابلیت رو داره؟

**Comment by AmirrezaNasiri** [comment]
سلام، وقتتون بخیر
فعلاً متاسفانه این امکان توی پکیج تعبیه نشده. به لیست فیچرهای آینده اضافه اش کردم :)
